packages/client/src/options.ts

```diff
@@ -7,20 +7,27 @@ import {QParamType} from './QueryApi'
 export interface ConnectionOptions {
   /** base host URL */
   host: string
+
   /** authentication token */
   token?: string
+
   /** token authentication scheme. Not set for Cloud access, set to 'Bearer' for Edge. */
   authScheme?: string
+
   /**
    * socket timeout. 10000 milliseconds by default in node.js. Not applicable in browser (option is ignored).
    * @defaultValue 10000
+   * @Deprecated: Please use more specific properties like WriteOptions.timeout.
    */
   timeout?: number
+
   /**
    * stream timeout for query (grpc timeout). The gRPC doesn't apply the socket timeout to operations as is defined above. To successfully close a call to the gRPC endpoint, the queryTimeout must be specified. Without this timeout, a gRPC call might end up in an infinite wait state.
    * @defaultValue 60000
+   * @Deprecated: Please use more specific properties like QueryOptions.timeout.
    */
   queryTimeout?: number
```

Comment:
I'm not so sure that this should be deprecated.  It already defines that it is for setting the queryTImeout.  It also eventually gets used in setting an `RpcOption` value when the query transport gets instantiated.  `QueryOptions.timeout` can still also be used, and since it is more specific, could take precedent.